src/spetlr/transformers/simple_sql_transformer.py

```diff
@@ -33,4 +37,18 @@ def process(self, df: DataFrame) -> DataFrame:
         ]
         df = df.select(col_choose)
 
+        # The sql server min date is 1753-01-01. If a timestamp column of the
```

Comment:
Create a test of the logic. Could be added here: https://github.com/spetlr-org/spetlr/blob/main/tests/cluster/sql/test_simple_sql_etl.py

Reply:
I couldn't find the test file for this module. Thanks for the hint
I will include the test there.

Reply:
I will approve when you have created the tests.